modules/serverless-admin-init-containers.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-admin-init-containers_{context}"]
+= Enabling init containers
+
+link:https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[Init containers] are specialized containers that are run before application containers in a pod. They are generally used to implement initialization logic for an application, which may include running setup scripts or downloading required configurations.
+
+[NOTE]
+====
+Init containers may cause longer application start-up times and should be used with caution for serverless applications, which are expected to scale up and down frequently.
+====
+
+You can enable the use of init containers for Knative services by adding the `kubernetes.podspec-init-containers` flag to the `KnativeServing` custom resource (CR).
```

Comment:
This is almost exactly the same as the procedure. You should delete it here. I don't think you'll lose anything by doing so.

Reply:
I will maybe revisit this either to add that adding this enables init containers to the step in the procedure or make this part of the initial paragraph in the intro.